cheatsheets/Vulnerable_Dependency_Management_Cheat_Sheet.md

```diff
@@ -231,9 +231,9 @@ It's important to ensure, during the selection process of a vulnerable dependenc
         * Full support: Java, .Net.
         * Experimental support: Python, Ruby, PHP (composer), NodeJS, C, C++.
     * [NPM Audit](https://docs.npmjs.com/cli/audit)
-        * Full support: NodeJS, Javascript.
+        * Full support: NodeJS, JavaScript.
         * HTML report available via this [module](https://www.npmjs.com/package/npm-audit-html).
-    * To manage the vulnerable dependencies in a transversal way in an organization the [OWASP Dependency Track](https://dependencytrack.org/) project can be used.
+    * To manage the vulnerable dependencies in a transversal (??) way in an organization the [OWASP Dependency Track](https://dependencytrack.org/) project can be used.
```

Comment:
I have no idea what this is trying to say, it's a really odd word. There's one use of `traversal` elsewhere in this corpus, but I can't say that's any better.

Reply:
I think this should be something like `To manage vulnerable dependencies across an organization the OWASP....`